security-release-process.md

```diff
@@ -0,0 +1,597 @@
+      - [Tampering](#tampering-3)
+  - [Glossary](#glossary)
+    - [Adjacent Network Access](#adjacent-network-access)
+    - [Authenticated User](#authenticated-user)
+    - [Local Access](#local-access)
+    - [Remote Anonymous User](#remote-anonymous-user)
+    - [High Value Asset](#high-value-asset)
+    - [Service Failure](#service-failure)
+
+## Security Response Committee (SRC)
+
+Security vulnerabilities should be handled quickly and sometimes privately. The primary goal of this process is to reduce the total time users are vulnerable to publicly known exploits.
+
+The [Security Response Committee (SRC)](README.md#security-response-committee-src) is responsible for organizing the entire response including internal communication and external disclosure but will need help from relevant developers and release managers to successfully run this process.
+
+The SRC will consist of volunteers subscribed to the private [Kubernetes Security](https://groups.google.com/a/kubernetes.io/forum/#!forum/security) list.
+
+### Security Response Committee Membership
+
+#### Joining
+
+New potential members to the SRC will first fill a minimum of a 3 month rotation in the [Associate](#Associate) role.
+
+These individuals will be nominated by individuals on steering committee, SRC, or release manager lead and patch release manager.
+
+The SRC has a goal to have at least 7 members.
+
+To encourage diversity members will also abide by a 1/2 maximal representation from any one company at any time. If representation changes due to job shifts or mergers, then members are encouraged to grow the team or replace themselves through mentoring new members. Being overlimit should not continue for longer than 12 months to give time to identify and mentor new members.
+
+##### SRC Lazy Consensus Selection
+
+Selection of new members from the associates will be done by lazy consensus.
+
+- Use lazy consensus amongst members for adding new people up to 7 from the pool of eligible associate members with fallback on majority vote.
+- Use lazy consensus amongst other members with fallback on majority vote to accept proposal on step down.
+
+#### Stepping Down
+
+Members may step down at anytime and propose a replacement from the pool of eligible associate members.
+
+#### Responsibilities
+
+- Members should remain active and responsive, and participate in the [oncall rotation](src-oncall.md).
+- Members taking an extended leave of 1 or more months should coordinate with other members to ensure the role is adequately staffed during the leave.
+- Members going on leave for 1-3 months may identify a temporary replacement.
+- Members of a role should remove any other members that have not communicated a leave of absence and either cannot be reached for more than 1 month or are not fulfilling their documented responsibilities for more than 1 month. This may be done through a super-majority vote of members.
+
+##### Incident Commander
```

Comment:
I'd like to make it easier for SRC members to delegate more IC responsibilities to community security contacts. See https://github.com/kubernetes/committee-security-response/issues/140